go/cmd/berty/omnisearch.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+
+	ff "github.com/peterbourgon/ff/v3"
+	"github.com/peterbourgon/ff/v3/ffcli"
+	"go.uber.org/zap"
+
+	"berty.tech/berty/v2/go/internal/config"
+	"berty.tech/berty/v2/go/internal/initutil"
+	"berty.tech/berty/v2/go/internal/omnisearch"
+	omniEngineBerty "berty.tech/berty/v2/go/internal/omnisearch/engines/berty"
+	omniParseBerty "berty.tech/berty/v2/go/internal/omnisearch/parsers/berty"
+	omniProvManager "berty.tech/berty/v2/go/internal/omnisearch/providers/manager"
+	omniProvRdvp "berty.tech/berty/v2/go/internal/omnisearch/providers/rdvp"
```

Comment:
this should not be imported in cmd, hide this in the omnisearch entrypoint package